utils/uploadFile.js

```diff
@@ -12,6 +12,8 @@ const uploadFile = async (url, cookie, filePath) => {
       Cookie: cookie,
       ...formData.getHeaders(),
     },
+    maxContentLength: Infinity,
```

Comment:
This might be dangerous if the server doesn't know how to handle things correctly. But let's keep it now. Later, with @ahgentil, let's try implementing the stream write.

Reply:
http://mongodb.github.io/node-mongodb-native/api-generated/cursorstream.html
I think this is the way: create a stream from query/cursor and pipe it into the post request to graasp.